boost.sh

```diff
@@ -1057,7 +1066,11 @@ deployToNexus()
         deployPlat "osx" "$BUILDDIR"
     fi
     if [[ -n "$BUILD_LINUX" ]]; then
-        deployPlat "linux" "$BUILDDIR"
+        if [[ -n "$USE_CXX11_ABI" ]]; then
```

Comment:
The proper support matrix here must be:
USE_CXX11_ABI==empty: non-ABI specifying build, deploy "linux"
USE_CXX11_ABI==1: deploy "linux", compatible with modern ABI
USE_CXX11_ABI==0: deploy "linux-cxx11-abi-disabled"

Note, that cxx11-abi-enabled is compatible with modern gcc versions (while the abi-disabled version is not)

Reply:
I can address the naming convention. Will CI builds for "linux" define `USE_CXX11_ABI==1`?

Reply:
Generally, no, it will be either 0 or empty.

Alternatively, rename it to `USE_PRE_CXX11_ABI` and invert the values:
`USE_PRE_CXX11_ABI=1` for "linux-cxx11-abi-disabled"
`USE_PRE_CXX11_ABI=0` or `USE_PRE_CXX11_ABI=` for "linux"